src/api/spec/controllers/staging/staged_requests_controller_spec.rb

```diff
@@ -99,6 +99,65 @@
       it { expect(staging_project.staged_requests).to include(bs_request) }
       it { assert_select 'status[code=ok]' }
     end
+
+    context 'with an excluded request', vcr: true do
+      context 'when not providing the remove exclusion parameter' do
+        before do
+          create(:request_exclusion,
+                 bs_request: bs_request,
+                 number: bs_request.number,
+                 staging_workflow: staging_workflow)
+          login user
+          post :create,
+               params: { staging_workflow_project: staging_workflow.project.name,
+                         staging_project_name: staging_project.name, format: :xml },
+               body: "<requests><number>#{bs_request.number}</number></requests>"
+        end
+
+        it { expect(response).to have_http_status(:bad_request) }
+        it { expect(response.body).to match(/invalid_request/) }
+        it { expect(response.body).to match(/Error while removing excluded requests: Request #{bs_request.number} not found in Staging for project #{staging_workflow.project.name}/) }
+      end
+
+      context 'when providing the remove exclusion parameter' do
+        before do
+          create(:request_exclusion,
+                 bs_request: bs_request,
+                 number: bs_request.number,
+                 staging_workflow: staging_workflow)
+          login user
+          post :create,
+               params: { staging_workflow_project: staging_workflow.project.name,
+                         staging_project_name: staging_project.name,
+                         remove_exclusion: true,
```

Comment:
I wonder if it's better to set '1' instead of 'true' :thinking:  All in all, '1' is the value in the query string.

Reply:
As we talked offline, we are just checking parameter presence, so we don't really care what's inside that parameter